components/SwipeRow.js

```diff
@@ -132,6 +171,10 @@ class SwipeRow extends Component {
 	}
 
 	handlePanResponderMove(e, gestureState) {
+		/* If the view is force closing, then ignore Moves. Return */
+		if(this.state.isForceClosing) return;
```

Comment:
Please add `{` `}` here

Reply:
Also.. if `forceCloseRow` is using `manuallyCloseRow` I don't think there's any need to `return` here. This shouldn't run

Reply:
@jemise111 Got it! That makes sense! I'll try that!

Reply:
@jemise111 Curly brackets `{` `}` are added! (https://github.com/jemise111/react-native-swipe-list-view/pull/299/commits/a31030d8401a7aaa1e2093d1b1421b19bd98266f). 

However, it seems that conditional return chunk is still needed.😢 

When the force-closing is happening, it seems the `handlePanResponderMove` is still in action due to the finger presence on the screen.(Yup. force closing!). And force-closing with `handlePanResponderMove` together makes the whole row to be gone, and after that the force-closing no longer gets trigged. Here are two GIF images. First one is with return, and the next one is without the return.


![ezgif com-optimize](https://user-images.githubusercontent.com/4941095/59994597-6f304680-9622-11e9-92df-2a169eb0374c.gif)

![ezgif com-optimize (1)](https://user-images.githubusercontent.com/4941095/59994714-e239bd00-9622-11e9-813c-017aaf5bf6bd.gif)